### PR TITLE
feat(git-practice): Diego's rehearsal

### PR DESCRIPTION
**Pull request by d-jv**
Created 2 txt files, one of them with no important info at all and the second one with my name inside of it.

### CODE REVIEW

**Review by RickyCode — COMMENTED** [review]
please change de name of the pull request to the format required

**Review by carrizotamara — COMMENTED** [review]
nice

File: diego-git-practice/file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Name: Diego
```
Comment (arielarmijo) [inline diego-git-practice/file2.txt]: please add last name

**Review by arielarmijo — DISMISSED** [review]

**Review by carrizotamara — DISMISSED** [review]

File: diego-git-practice/file2.txt (hunk 1 above)
Comment (mframirezfuentes) [reply diego-git-practice/file2.txt]: please, put your profession

Comment (d-jv) [reply diego-git-practice/file2.txt]: > please add last name

Added !

> please, put your profession

Added !

**Review by carrizotamara — APPROVED** [review]

**Review by RickyCode — APPROVED** [review]

**Review by arielarmijo — APPROVED** [review]

**Pull request merged**